keyboards/dp3000/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

Comment:
Encoder map should only be enabled at the keymap level, not the keyboard level.  And you may want/need a rules.mk with the setting enabled for the default keymap too.
```suggestion
```

Reply:
Suggestion changes, resolved on new commit. Thank you